scripts/verify-dependency-licenses.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+change_dependency_licenses=$(git status | grep dependency_licenses.txt | wc -l)
+
+if [[ change_dependency_licenses -gt 0 ]]; then
+  echo "Detected license change. Please run 'yarn extract-dependency-licenses' and add dependency_licenses.txt file changes to the change set."
+  git status | grep API.md
```

Comment:
```suggestion
  git status | grep dependency_licenses.txt
```

Not really sure this adds much value to the verify API or dependency licenses scripts because the echo above already shows what is wrong.

Reply:
Yeah, in addition to it being wrong, the diff may be sizeable. I'll yank. Thank you!